lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -2,38 +2,42 @@
 
 exports[`Performance: metrics evaluates valid input correctly 1`] = `
 Object {
-  "estimatedInputLatency": 78,
+  "estimatedInputLatency": 13,
```

Comment:
I think we need a much better trace if we're going to be replacing our existing one. the existing one has distinguished moments for each metric and lots of substance to the main thread examinations which is important to preserve to catch changes. when they all devolve to FMP graph we're not really exercising very much of the lantern logic.

WDYT about just adding one for LCP for now and we can reevaluate a new golden trace file to use for everything once we do analysis?

Reply:
also would be nice not to have a ton of churn in all the golden LHR stuff too

Reply:
sounds good to me.

But the lcp metric will error in `metrics-test` if lantern is used on the old trace. Do we want to not throw an error if LCP can't be figured out?

Reply:
> But the lcp metric will error in metrics-test if lantern is used on the old trace. Do we want to not throw an error if LCP can't be figured out? Or use the m79 trace for just that test, and update the snapshots?

Maybe keep throwing (it'll be useful to be an error in the lcp audit when collecting data) but catch just for LCP (until we've collected data, validated, etc) in metrics.js?

Reply:
+1 to brendan's suggestion, I believe we already catch for TTI anyhow there